Kitodo/src/test/java/org/kitodo/DummyStructuralElementView.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import org.kitodo.api.dataeditor.rulesetmanagement.DatesSimpleMetadataViewInterface;
+import org.kitodo.api.dataeditor.rulesetmanagement.Domain;
+import org.kitodo.api.dataeditor.rulesetmanagement.MetadataViewInterface;
+import org.kitodo.api.dataeditor.rulesetmanagement.MetadataViewWithValuesInterface;
+import org.kitodo.api.dataeditor.rulesetmanagement.StructuralElementViewInterface;
+
+public class DummyStructuralElementView implements StructuralElementViewInterface {
```

Comment:
Why are you creating new implementations, which are doing nothing? They should be used by the method you test

Reply:
`DummyStructuralElementView` **is used** from `DummyRulesetManagement`, which in turn is used in `DummyRulesetManagement.testBuildStructureTreeRecursively()`.

Reply:
But why are you using dummyImplementations instead of the original ones, which are actually used in productive code?

Reply:
Because it would be very complicated then. You would need a full setup, ruleset, processes, etc.
I support the idea of putting the meta-data editor under tests, but such a big development should be done in a separate branch (or even several), not within a small changeset which is to display links.

Reply:
I do not believe that tests will be added later. We all know how reality looks. If tests are not inserted parallel to the implementation, they are not inserted at all.

Reply:
You do remember, that our MockDatabase is providing a full setup, right?

Reply:
Not for the scenario. You could add one more in MockDatabase, but like I said: Not in a day.